apps/docs/.vitepress/plugins/utils/extractImports.test.ts

```diff
@@ -86,6 +86,36 @@ describe('extractImports', () => {
     });
   });
 
+  describe('validateSnippetContent', () => {
+    it('should pass validation for a valid code snippet content', () => {
+      const codeSnippet = [
+        '// #import { AssetId };',
+        '',
+        'const assetId: AssetId = {',
+        '  value: Bits256,',
+        '};',
+      ];
+      const filepath = '/some/file/asset-id.test.ts';
+      expect(() => validateSnippetContent(codeSnippet, filepath)).not.toThrow();
+    })
+
+    it('should throw an error when malformed #imports detected', async () => {
+      const codeSnippet = [
+        '// #import { AssetId }',
+      ]
+      const filepath = '/some/file/asset-id.test.ts';
+      
+      await expectToThrowFuelError(
+        () => validateSnippetContent(codeSnippet, filepath),
+        new FuelError(
+          ErrorCode.VITEPRESS_PLUGIN_ERROR,
+          `Found malformed "#import" statements in code snippet.\n\nPlease check "${filepath}".\n\n// #import { AssetId }`
```

Comment:
Is it worth stating it needs to include semi colons?

Reply:
Potentially, would have thought the warning would prompt the user - maybe add the correct format before listing out the imports?

Reply:
Agreed but I think that would be helpful

Reply:
9303dff1f061ebf65e51e6ab035413b5171a552f